sample-apps/react/egress-composite/src/components/LogoAndTitle.scss

```
@@ -1,17 +1,18 @@
-.eca__logo-and-title-overlay {
+@layer base-layer {
+  .eca__logo-and-title-overlay {
     position: absolute;
     display: grid;
     top: 0;
     right: 0;
-  
+
     width: 100%;
```

Stylelint (17.7.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)